src/index.js

```diff
@@ -10,6 +10,8 @@ const EXCLUDE_PATHS = new Set([
 	'tenant', 'org', 'initialServerlessConfig'
 ]);
 
+const KEY_REGEXP = /^\$forEach$|^\$forEach_.+$/;
```

Comment:
This will allow whitespace and other potentially bad things in the suffix, but I assume things would have already failed before we even get to this point?

Reply:
Yeah, I think yaml would not allow that, but I can definitely switch to `\S` instead of `.`

Reply:
Eh, this is probably fine.